packages/cli/medusa-cli/src/create-cli.ts

```diff
@@ -125,6 +125,28 @@ function buildLocalCommands(cli, isLocalProject) {
       desc: `Create a new Medusa project.`,
       handler: handlerP(newStarter),
     })
+    .command({
+      command: `db:create`,
+      desc: `Manage migrations from the core and your own project`,
```

Comment:
todo: Update the description to reflect the new command

Reply:
haha. Missed it :)